lib/const.rb

```diff
@@ -1,5 +1,5 @@
 # Defines common constants used in different parts of crew
-CREW_VERSION = '1.36.3'
+CREW_VERSION = '1.36.5'
```

Comment:
Only need to bump to `.4`?

Reply:
Intentional to fixup people testing...

Reply:
see new changes.

Reply:
Who tests other than me or you?  Are we overthinking this? :)